### PR TITLE
Change title, ack individuals

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

**Comment by stpeter** [comment]
I like the new title - short and sweet.

**Comment by richsalz** [comment]
I think it's also a nod to the older days when RFC's had titles like "On holy wars..." and such :)

File: draft-ietf-uta-rfc6125bis.md [hunk 1]
```diff
@@ -13,9 +13,7 @@ pi:
   toc: 'yes'
   tocdepth: '4'
   rfcedstyle: 'yes'
-title: Representation and Verification of Domain-Based Application Service Identity
-  within Internet Public Key Infrastructure Using X.509 (PKIX) Certificates in the
-  Context of Transport Layer Security (TLS)
+title: On TLS Service Names
```
Comment (martinthomson) [inline draft-ietf-uta-rfc6125bis.md:16]: Or
```suggestion
title: Service Names in TLS
```

Comment (stpeter) [reply draft-ietf-uta-rfc6125bis.md:16]: No strong preference here. Martin's might be slightly clearer.

**Pull request merged**